src/slic3r/GUI/Preferences.cpp

```diff
@@ -84,6 +84,14 @@ void PreferencesDialog::build()
 	option = Option (def, "preset_update");
 	m_optgroup_general->append_single_option_line(option);
 
+	// Please keep in sync with ../Utils/PresetUpdater.cpp
+	def.label = L("Allow updating Presets from non-Prusa sites");
+	def.type = coBool;
+	def.tooltip = L("If enabled, Slic3r will also allow updates of built-in system presets from URLs other than the official Prusa URL.");
```

Comment:
Shouldn't it read "Prusa Slicer" instead of "Slic3r"?

Reply:
Quite possibly. I chose "Slic3r" to follow the convention used in Preferences.cpp. Pretty much every preference codes its tooltip with "Slic3r" as well. I'm not sure how they show up as "PrusaSlicer" in the app.